src/common/options.cc

```diff
@@ -8640,6 +8640,40 @@ std::vector<Option> get_mds_client_options() {
         "If no filesystem name is given in mount() or this setting, the default "
         "filesystem will be mounted (usually the first created)."),
 
+    // MSGR On-wire compression
+
+    Option("ms_osd_compress_mode", Option::TYPE_STR, Option::LEVEL_ADVANCED)
+    .set_default("force")
+    .set_enum_allowed({"none", "force"})
+    .add_service("osd")
+    .set_flag(Option::FLAG_RUNTIME)
+    .set_description("Compression policy to use in MSGR for communicating with OSD")
```

Comment:
is MSGR defined anywhere in the document? is this an abbreviation ? how is enduser supposed to interpret this?

Reply:
Changing to `Messenger`